cmd/login.go

```diff
@@ -103,16 +103,27 @@ func runOCMWorkspaceContainer(
 	serviceRef string,
 	isOcmLoginOnly bool) {
 	envVarOcmUser := fmt.Sprintf("OCM_USER=%s", viper.GetString("ocUser"))
-	envVarOcmToken := fmt.Sprintf("OCM_TOKEN=%s", viper.GetString("ocmToken"))
+	// envVarOcmToken := fmt.Sprintf("OCM_TOKEN=%s", viper.GetString("ocmToken"))
 	envVarCluster := fmt.Sprintf("OCM_CLUSTER=%s", ocmCluster)
 	envVarIsOCMLoginOnly := fmt.Sprintf("IS_OCM_LOGIN_ONLY=%v", isOcmLoginOnly)
-	userHome := fmt.Sprintf("/home/%s", viper.GetString("ocUser"))
+	userHome := fmt.Sprintf("%s", viper.GetString("userHome"))
+
+	// Fetch the latest OCM_TOKEN
+	out, err := exec.Command("ocm", "token").Output()
```

Comment:
Can you please create this function in utils.go? I think this will be reused in many places.